lang/step.md

```diff
@@ -28,7 +28,9 @@ impl<M: Memory> Machine<M> {
         } else {
             // Bump up PC, evaluate this statement.
             let stmt = block.statements[next_stmt];
-            self.cur_frame_mut().next_stmt = next_stmt + 1;
```

Comment:
`frame.next_stmt` does not exist anymore, it's `next: (BbName, BigInt)` now apparently.

Reply:
Ah right my last commit here was buggy.